ui/packages/consul-ui/app/templates/dc/services/show/topology.hbs

```diff
@@ -25,6 +25,15 @@ as |route|>
       {{#if topology.FilteredByACLs}}
         <TopologyMetrics::Notice::LimitedAccess />
       {{/if}}
+      {{#if topology.DefaultAllow}}
+        <TopologyMetrics::Notice::DefaultAllow />
+      {{/if}}
+      {{#if topology.WildcardIntention}}
+        <TopologyMetrics::Notice::WildcardIntention />
+      {{/if}}
+      {{#if topology.undefinedIntention}}
+        <TopologyMetrics::Notice::UndefinedIntention />
+      {{/if}}
```

Comment:
Components/partials are usually 'write something once, and use multiple times' I think here we've done the reverse in that we have written multiple components and used each one a single time.

Is there a way we can do this less repetitively, especially if all the copy is done with `t`?

Reply:
I tried. It all comes down to making the footer dynamic. I ran into some issues and just left them split up.